app/controllers/performance_reports_controller.rb

```diff
@@ -0,0 +1,67 @@
+class PerformanceReportsController < ApplicationController
+  include MakeNotice
+
+  before_action :set_performance_report, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @performance_reports = PerformanceReport.all
+    authorize PerformanceReport
+  end
+
+  def show; end
+
+  def new
+    @performance_report = PerformanceReport.new
+    authorize @performance_report
+  end
+
+  def edit
+    @performance_report.period_start = @performance_report.period_start.to_date
+    @performance_report.period_end = @performance_report.period_end.to_date
```

Comment:
Why is `.to_date` necessary here? Maybe we can just use `date` instead of `datetime` in the migration?

Reply:
Yes, that has definitely quite something to it. I guess I got here, because of the assumption I made, when I set up the thing. If I had Assignment end and assignment start as datetime, whould an edge-case 31.12.xx 15:12 field be associated with a report period end 31.12.xx ?

Reply:
quite possible that that was an unnecessary bloat I made here.